ui/ui-frontend/projects/collect/src/app/collect/projects/create-project/create-project.component.ts

```diff
@@ -455,7 +456,7 @@ export class CreateProjectComponent implements OnInit, AfterViewChecked {
     this.moveToNextStep();
     const zipFile = new ZipFile();
     this.zipFileStatus$ = zipFile.zipFileStatus$;
-    if (this.importType === ImportType.COMPRESSED) {
+    if (this.importType !== ImportType.DIRECTORIES_FILES) {
```

Comment:
Pour une meilleure maintenabilité (quand on ajoutera d'autres import types par la suite) :
```suggestion
    if ([ImportType.COMPRESSED, ImportType.SIP].includes(this.importType as ImportType)) {
```

Reply:
Modifié @marob